### ISSUE TITLE
Automatically download subtitles

### ISSUE THREAD

**Issue by rossburton** [media-explorer/media-explorer#233]
A button somewhere to automatically download subtitles for the currently playing video would be really useful.

**Comment by semiot** [comment]
Yes, it would, but where would you automatically download subtitles from?

**Comment by rossburton** [comment]
Both It's Playing on iOS and Totem on GNOME can download from opensubtitles.org.

The Totem plugin -- albeit in Python -- is probably a good starting point.

**Comment by hadess** [comment]
I'm happy to see any code in Totem replaced by C code, and we're willing to share!
There's also a patch in Totem's bugzilla to support other websites and services.
